gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/controller/LoginController.java

```diff
@@ -59,18 +65,16 @@ public class LoginController {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LoginController.class);
     private static final String LOGIN_VIEW = "login";
-    private static final Map<String, String> socialProviderTypes;
-    static {
-        Map<String, String> sMap = new HashMap<>();
-        sMap.put("github-am-idp", "github");
-        sMap.put("google-am-idp", "google");
-        sMap.put("twitter-am-idp", "twitter");
-        sMap.put("facebook-am-idp", "facebook");
-        sMap.put("franceconnect-am-idp", "franceconnect");
-        sMap.put("azure-ad-am-idp", "microsoft");
-        sMap.put("linkedin-am-idp", "linkedin");
-        socialProviderTypes = Collections.unmodifiableMap(sMap);
-    }
+    private static final Duration SOCIAL_IDP_STATE_EXPIRES_AFTER = Duration.ofMinutes(5);
```

Comment:
5 minutes is probably too short if the IDP is configure to use factors like email or mobile app validation. I would define it to 15 minutes and I would make it configurable using a @Value (maybe the key could be `security.stateDuration` in millis)